client/packages/sdk/package.json

```diff
@@ -40,6 +40,7 @@
   "dependencies": {
     "@polkadot/api": "^8.9.1",
     "dotenv": "^16.0.3",
+    "interfaces": "link:@polkadot/types/interfaces",
```

Comment:
Why link?

Reply:
Was created by default when installing the dependencies

Reply:
Pin it to a version

Reply:
just updated it